README.md

```diff
@@ -1,41 +1,131 @@
-# ORAS Documentation
 
-[![Netlify Status](https://api.netlify.com/api/v1/badges/db61db6e-a953-4b02-b5fb-7f04f018b9d8/deploy-status)](https://app.netlify.com/sites/oras-project/deploys)
+<img src="https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png" alt="FLINT UI logo" height ="auto" width="200" />
+<br />
 
-![ORAS](https://github.com/oras-project/oras-www/raw/main/docs/assets/images/oras.png)
+<h1>ORAS Documentation</h1>
+ 
+An ORAS UI is written in Hugo Framework, to provide an awesome user interface for configuring simulations.
```

Comment:
It doesn't use the Hugo at the moment.

Reply:
@TerryHowe sir, Do you have some idea about the content that we can add here instead of this? Does that give a clear-cut view of the project to the new contributors?

Reply:
Look around I don't really seen anything that is concise. You could say something generic like "This is the web site and documentation for the ORAS project."

Reply:
@TerryHowe Done with the changes suggested by you.
Let me know if anything I need to change
Otherwise we can merge this PR.